src/main/java/net/sf/jabref/model/ChainNode.java

```diff
@@ -126,6 +126,7 @@ protected void setParent(T parent) {
      *
      */
     public void removeChild() {
+        // NPE if this is ever called
         child.setParent(null);
```

Comment:
The first comment was not really relevant here. Again, the thing is not that `child` becomes `null`, although that will also break the code, but that whenever `child.setParent(null)` is called there will be an NPE as the code for `setParent`reads (second comment):
```
     protected void setParent(T parent) {
         this.parent = Objects.requireNonNull(parent);
}
```
Hence, yes, I can add the if statement, but that is not really solving the problem I pointed out in the comment.